libs/vkd3d/memory.c

```diff
@@ -914,18 +916,21 @@ static HRESULT vkd3d_memory_allocator_flush_clears_locked(struct vkd3d_memory_al
     while (queue_mask)
     {
         queue_index = vkd3d_bitmask_iter32(&queue_mask);
-        queue = device->queues[queue_index];
+        queue_family = device->queue_families[queue_index];
 
-        if (!(vk_queue = vkd3d_queue_acquire(queue)))
-            return E_FAIL;
+        for (i = 0; i < queue_family->queue_count; i++)
```

Comment:
I think we need to move to a model where submission threads wait on the allocator timeline on-demand rather than broadcasting to ~10+ queues every time we zero memory. Could make this a TODO.

Reply:
Foxed.

Reply:
🦊